### PR TITLE
update(OWNERS): move inactive maintainers to emeritus approvers

### PR DESCRIPTION
**Pull request by jasondellaluce**
**What type of PR is this?**

/kind documentation

**Any specific area of the project related to this PR?**

**What this PR does / why we need it**:

For https://github.com/falcosecurity/evolution/issues/157 (see: https://github.com/falcosecurity/evolution/issues/157), this PR proposes to move to `emeritus_approvers` maintainers that have been inactive in the past 6 months, as for our governance (see: https://github.com/falcosecurity/.github/blob/master/GOVERNANCE.md#project-inactivity). Maintainers are invited to have a discussion and take a decision.

cc @falcosecurity/community-maintainers 

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by leogr** [comment]
Although this repository has not required much involvement, I'm ok with moving those owners to emeritus approvers because they egregiously served the Falco project in the past, but now it seems they don't have time anymore.

However, I don't feel comfortable remaining the only approver on the list. So I want to call in this discussion some other active maintainers I know have been very present in the community.

@araujof @terylt @cpanato @maxgio92 @Issif @dwindsor: Is anyone willing to step up as a maintainer of this repository? :hugs: 

Ofc, the invite is also open to any maintainer interested in managing this repo :smiley:

**Comment by Issif** [comment]
I'll submit myself as maintainer in a next community call if people agree

**Comment by Andreagit97** [comment]
Hey, @leogr if you need help, I can do it! Let me know if you need it :)

**Comment by terylt** [comment]
I'm happy to help maintain this if you want.

**Comment by maxgio92** [comment]
I'm happy to help here too @leogr. At the same time, as community calls are one of the best places where the whole community meets IMHO, and I join them infrequently, I don't see I match the duties of a potential community maintainer.

**Comment by araujof** [comment]
@leogr same here, if need one more pair of hands, I volunteer to help :)

File: OWNERS [hunk 1]
```diff
@@ -1,19 +1,13 @@
 approvers:
-  - leodido
-  - fntlnz
-  - mfdii
-  - kris-nova
   - leogr
```
Comment (leogr) [inline OWNERS:2]:
```suggestion
  - leogr
  - Issif
  - Andreagit97
  - terylt
  - maxgio92
  - araujof
```

I believe we have already reached a good number of maintainers here. Note that this is a special repository, and it won't probably need too much maintenance. However, any idea to improve the repository is always welcome!

PS if other maintainers still want to join, please propose yourself! (even after this PR gets merged)

**Review by leogr — COMMENTED** [review]
Hey @jasondellaluce,

Could you apply my suggested change (see below) please? Thank you :pray: 

Then I believe we are mostly done here.

**Review by leogr — COMMENTED** [review]
This change LGTM since:

- people moved to `emeritus_approvers` have been inactive but also 
  - decided to step down, 
  - have not shown interest in maintaining this repo,
  - are unreachable,
  - or already left the organization;
- new people added to `approvers`
  - are already maintainers of other projects
  - have been involved in the community;
- this is a special repo, but it does not require much maintenance, and no particular requirements are needed.

I've added the `wip:` prefix (i.e. work in progress) to this PR to make clear that the current proposed change is not yet a definitive decision. So, I leave this open another couple of days in case there's other feedback. However, you can still propose changes even after this PR gets merged.

**Pull request merged**